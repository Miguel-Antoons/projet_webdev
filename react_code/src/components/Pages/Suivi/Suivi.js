import React from 'react';

function Suivi() {
    return (

        <h1>bienvenue dans Suivi</h1>

    )

}


export default Suivi;