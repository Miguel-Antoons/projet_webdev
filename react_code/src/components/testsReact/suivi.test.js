//eteindre react : ctrl c
//npm run test
//npm i --save-dev @testing-library/react react-test-renderer

import { render, screen, cleanup } from '@testing-library/react'
import Suivi from '../Pages/Suivi/Suivi'
import renderer from 'react-test-renderer';


test('test', () =>{
    expect(true).toBe(true);
})

//test('devrais montrer le component suivi', () =>{
    //render(<Suivi/>);
    //const suiviElement = screen.getByText('<body>');
    //expect(suiviElement).toBeInTheDocument();
//})

test('render bien la page', () => {
    const page = renderer.create(<Suivi/>).toJSON();
    expect(page).toMatchSnapshot();
  });

