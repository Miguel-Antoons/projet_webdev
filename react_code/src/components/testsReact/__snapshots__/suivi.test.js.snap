// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render bien la page 1`] = `
<div
  className="container"
>
  <div
    className="suivi jumbotron"
  >
    <h1>
      Suivi du Matériel
    </h1>
  </div>
  <table
    className="table"
  >
    <thead>
      <tr>
        <th>
          ID
        </th>
        <th>
          Matériel
        </th>
        <th>
          Nom
        </th>
        <th>
          Téléphone
        </th>
        <th>
          Email
        </th>
        <th>
          Date Emprunt
        </th>
        <th>
          Rendu
        </th>
      </tr>
    </thead>
    <tbody
      id=""
    />
  </table>
</div>
`;
